MssBuilder/MssValueTypeBuilder.cs

```diff
@@ -0,0 +1,78 @@
+using HandlebarsDotNet;
+using Mss;
+using Mss.Types;
+using System.Diagnostics;
+
+namespace MssBuilder
+{
+    public class MssValueTypeBuilder
+    {
+        private readonly string _solutionPath;
+        public readonly string ProjectName = "ValueTypes";
+        public readonly string ProjectPath;
+
+        private readonly string _valueClassTemplate =
+@"namespace {{Namespace}}
+{
+    class {{ClassName}}
+    {
+        public {{TypeName}} {{PropName}} { get; set; }
+
+        public {{ClassName}}() { }
+        public {{ClassName}}({{TypeName}} value) => {{PropName}} = value;
+    }
+}";
```

Comment:
Possible to move this to Template folder and use TemplateFactory?

Reply:
moved into templates